models/__init__.py

```diff
@@ -0,0 +1,279 @@
+# coding=utf-8
+import pandas as pd
+from sqlalchemy import func
+from models.cyberconnect2 import Session, engine
+
+from models.t_ice_time import Ice_Time
+from models.t_locations import Locations
+from models.t_icetype import IceType
+from models.t_coaches import Coaches
+from models.t_equip import uSkateConfig, uSkaterBlades, uSkaterBoots
+
+
+session = Session()
+
+
```

Comment:
I've created a separate issue, #5 to consider how to go about structuring this file and the classes within it.  Its very unlikely that the final product will resemble the current state since most of these statements were used to verify table and code structure while learning.

There are many opportunities for improvement here.